src/modules/content-s3/content-s3.c

```diff
@@ -500,17 +500,16 @@ int mod_main (flux_t *h, int argc, char **argv)
     if (content_register_backing_store (h, "content-s3") < 0)
         goto done;
     if (content_register_service (h, "content-backing") < 0)
-        goto done;
+        goto done_unreg;
     if (content_register_service (h, "kvs-checkpoint") < 0)
-        goto done;
+        goto done_unreg;
     if (flux_reactor_run (flux_get_reactor (h), 0) < 0) {
         flux_log_error (h, "flux_reactor_run");
-        goto done;
+        goto done_unreg;
     }
-    if (content_unregister_backing_store (h) < 0)
```

Comment:
I think this is already done automatically (unregistering services on module unload).
However it doesn't hurt to do it explicitly.